### PR TITLE
feat: added accordion button to active toolbar

### PR DESCRIPTION
**Pull request by jasminepawliw**
# [ZUDF-151](https://zucommunications.atlassian.net/browse/ZUDF-151?atlOrigin=eyJpIjoiYTI1ZWMxYzYwZjQ2NDk2MGE3M2UwNDEwZmVlNDQ2MmMiLCJwIjoiaiJ9) Update

Updated previous PR to allow for accordion button as seen [here].(https://github.com/ZuCommunications/rapidkit_core/pull/4#issuecomment-2386806484)

@msabares Let me know if this looks okay! I will update if necessary

[ZUDF-151]: https://zucommunications.atlassian.net/browse/ZUDF-151?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/install/editor.editor.basic_html.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ dependencies:
     - filter.format.basic_html
   module:
     - ckeditor5
+_core:
+  default_config_hash: cWH0XZ10nENYbkU1shUGnajX6KGkP2pA-VLngjb9SSM
```
Comment (msabares) [inline config/install/editor.editor.basic_html.yml]: We'll need to yeet this. This makes this config specific to a drupal install instance (i think 🤷‍♂️ ). When this config gets copied over, this key/value will get generated automatically

**Review by msabares — APPROVED** [review]
Looks good, just need to remove that one key/value pairing and it can be merged!

File: config/install/filter.format.basic_html.yml [hunk 2]
```diff
@@ -10,56 +10,69 @@ dependencies:
     - linkit
     - media
     - pathologic
+_core:
+  default_config_hash: gp3vIAoN8UVw4TrNz_thZe0i66M64VK-zGpVU8Vv7k8
```
Comment (msabares) [inline config/install/filter.format.basic_html.yml]: Yeet this too!

**Comment by jasminepawliw** [comment]
Thanks @msabares :) consider those lines yeeted

**Pull request merged**